### ISSUE TITLE
Media Browser - add Copy Url 

### ISSUE THREAD

**Issue by FerX** [modxcms/revolution#13618]
### Summary
In Media Browser, menu right click of file, add the option to copy absolute url of file.

### Environment
MODX 2.5.8

![image](https://user-images.githubusercontent.com/4459841/30899497-61c5187e-a360-11e7-8a37-940447ff687f.png)

### Related
#11666 
#14078

**Comment by OptimusCrime** [comment]
Do you mean full file path? Or the URL? If the latter, how should we differentiate between contexts?

**Comment by FerX** [comment]
you could choose with a submenu which context,
or take that of the current resource (in case of using it via TV)

**Comment by rainbowtiger** [comment]
Yes, please! URL would be great. I have clients ask me all the time for the URLs of images and PDFs, and there is no simple way I can give them to find this out for themselves.

**Comment by sebastian-marinescu** [comment]
Exactly what @rainbowtiger said!
This just hit me (again), as my client also doesn't know how to deduce the URL of a file..
There is already the option "Copy path", so "Copy URL" would be extremely good here. 
Also, this need the base-path from the media-source and has nothing to do with contexts.

**Comment by alroniks** [comment]
@JoshuaLuckers I guess it solved by https://github.com/modxcms/revolution/pull/14078, isn't it?

**Comment by JoshuaLuckers** [comment]
> @JoshuaLuckers I guess it solved by https://github.com/modxcms/revolution/pull/14078, isn't it?

This is about copying the URL, not the path

**Comment by OlegAnTo2000** [comment]
now there is a button "open file url", it is very close, but I would like to add "copy url", thanks

**Comment by SnowCreative** [comment]
> now there is a button "open file url", it is very close, but I would like to add "copy url", thanks

Where is that button? I don't see one.

**Comment by smg6511** [comment]
Note that the addition of this feature would be highly unlikely for the 2.x branch, as it's in maintenance mode. It could be requested for 3.x. That said, if _anything_ is to make it into any branch, we need folks willing to test open PRs. Please get involved on that end if you can ;-)

**Comment by OlegAnTo2000** [comment]
> this feature would be highly unlikely for the 2.x branch, as it's in maintenance mode. It could be requested for 3.x

Do you mean I need to create a new issue in 3.x? I'm currently using mostly modx3. I just wanted to write in this topic because it is relevant for version 3 and so that it becomes more visible to developers

**Comment by OlegAnTo2000** [comment]
> Where is that button? I don't see one.

Now it's in the file tree, but it's still not in the media manager. 

<img width="361" height="203" alt="Image" src="https://github.com/user-attachments/assets/97aeeda0-0dc2-4382-97f2-4432db6038ba" />

**Comment by SnowCreative** [comment]
> Now it's in the file tree, but it's still not in the media manager.

OK. Should be caps though, "URL".
